Remotion/ObjectBinding/Web/Res/Themes/NovaViso/HTML/BocCheckBox.css

```diff
@@ -2,39 +2,63 @@
 
 .bocCheckBox
 {
-  margin-right: 0.5em;
+  display: inline-flex;
+  align-items: center;
+}
+
+.bocCheckBox input
+{
+  position: absolute;
+  z-index: -1;
+}
+
+.bocCheckBox span.description
+{
+  display: inline-flex;
+  align-items: center;
+}
+
+.bocCheckBox img,
+.bocCheckBox input ~ span.image-visualizer::before
+{
+  margin: var(--remotion-themed-padding);
   vertical-align: middle;
-  display: inline-block;
-  white-space: nowrap;
+  height: 1.5em;
+  width: 1.5em;
 }
 
-.bocCheckBox.readOnly
+.bocCheckBox input ~ span.image-visualizer::before
 {
+  visibility: visible;
+  background-image: url('../Image/sprite.svg#CheckBoxFalse');
+  display: inline-block;
+  content: "";
+  z-index: 1;
+  position: relative;
```

Comment:
```suggestion
```
It looks like we can drop this one or is this meant as the `relative` to the input's `absolete`? But since absolute/relative is usually done with parent/child relations, this seems incorrect / redundant to me?